eamena/eamena/templates/views/forms/location.htm

```diff
@@ -387,7 +393,12 @@ <h5 id="arches-subjects-form" class="section-headline">{% trans "Overall Site Sh
                         <div data-bind='foreach: getBranchLists()'>
                             <div class="arches-crud-record">
                                 <a href="javascript:void(0)" class="arches-CRUD-child" title="{% trans "Delete this value" %}"><i data-bind="click: $parent.deleteItem.bind($parent)" class="arches-CRUD-delete fa fa-times-circle"></i></a>
-                                <a href="javascript:void(0)" class="arches-CRUD-child arches-CRUD-edit" data-bind="click: $parent.editItem.bind($parent)" title="{% trans "Edit this value" %}"><span data-bind='{text: nodes.get("SITE_OVERALL_SHAPE_TYPE_NEW.E55", "label")}' style="padding-right:2px;"></span>
+                                <a href="javascript:void(0)" class="arches-CRUD-child arches-CRUD-edit" data-bind="click: $parent.editItem.bind($parent)" title="{% trans 'Edit this value' %}">
+                                {% if resourcetypeid == 'HERITAGE_FEATURE.E24' %}
+                                <span data-bind='{text: nodes.get("SITE_OVERALL_SHAPE_TYPE.E55", "label")}' style="padding-right:2px;"></span>
+                                {% else %}
+                                <span data-bind='{text: nodes.get("SITE_OVERALL_SHAPE_TYPE_NEW.E55", "label")}' style="padding-right:2px;"></span>
+                                {% endif %}
```

Comment:
@mradamcox  the _NEW tag will disappear at the end of the data migration process (using the rename_entity_type utility: https://github.com/azerbini/eamena_dev/blob/15561846fab51b0a8dc89aebd3240d8e41482c26/arches/management/commands/package_utils/migrate_resources.py#L385) so this bit of code will become redundant then.